src/containers/components/JobList/index.test.tsx

```diff
@@ -0,0 +1,16 @@
+import {render} from "@testing-library/react";
+import { Provider} from "react-redux";
+import { BrowserRouter } from "react-router-dom";
+import  JobList  from ".";
+import { store } from "../../store";
+describe('With React Testing Library', () => {
+  it("should render JobContent", () => {
+    const { container } = render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <JobList/>
+      </BrowserRouter>
+      </Provider>);
+    expect(container).toBeInTheDocument();
```

Comment:
It should be more detailed

Such as:
- Which text is in the document?
- Which component is in the document?
- ...